### ISSUE TITLE
The v1.10.7 introduced a couple of breaking changes: Comparison vs v1.10.5

### ISSUE THREAD

**Issue by davidpelayo** [dkfbasel/vuex-i18n#104]
I'm checking the differences of the `v1.10.5` and `v1.10.7` due I had translations using multiple placeholders that they were getting translated correctly before, and after upgrading they fail.

The comparison link, between those two versions, is the following: https://github.com/dkfbasel/vuex-i18n/compare/v1.10.5...v1.10.7.

So, were before this was correctly working:

```js
computedProp() {
  const quantityClassName = this.$style[`${this.mainClassname}__quantity`];
  const translationParameters = {
    0: `<span id="${this.fromId}" class="${quantityClassName}">${from}</span>`,
    1: `<span id="${this.toId}" class="${quantityClassName}">${to}</span>`,
    2: `<span id="${this.totalId}" class="${quantityClassName}">${total}</span>`,
  };

  return this.$t(resultInfoKey, translationParameters);
}
```

With the version `v1.10.7` returns the following
<img width="545" alt="screen shot 2018-12-24 at 10 18 14 am" src="https://user-images.githubusercontent.com/84540/50395564-3ee81800-0765-11e9-8164-68c3ca5484b3.png">

`Not all placeholders found`.

And I'm starting to suspect the root cause of this issue is at [this line of code](https://github.com/dkfbasel/vuex-i18n/compare/v1.10.5...v1.10.7#diff-7278dab49dd6e4424f694900f20af101R693). Which actually relates to #98 and #99.

Please, any feedback or fix, is welcome.

**Comment by tikiatua** [comment]
HI @davidpelayo,

This issue is quite likely due to the changed regular expression match that we use to identify placeholders in translation strings. With the new regex matcher, we require at least two characters, which breaks your code if the variable replacement results in a matcher with only one character (i.e. a number). 

I will adapt the regex matcher to also include matches for single characters as long as they match [A-Za-z0-9_]. Stay tuned.

**Comment by tikiatua** [comment]
should be fixed in latest release v1.10.9

**Comment by davidpelayo** [comment]
Additionally, to my message above, the latest version fails on IE11 due short hand properties not being transpiled, such us [this](https://github.com/dkfbasel/vuex-i18n/blob/6f3043670dd220437451277dd60e3b48b1771730/src/vuex-i18n-store.js#L17).

Where:
```js
SET_LOCALE(state, payload) {
}
```

It should be:
```js
SET_LOCALE: function(state, payload) {
}
```

![i18nerror](https://user-images.githubusercontent.com/84540/50589239-23c48900-0e86-11e9-8d14-99d452ac5e2f.gif)


Is this intended?

**Comment by tikiatua** [comment]
Hi @davidpelayo 

Thank you for reporting this. Definitely not intended. I will look into this.

**Comment by tikiatua** [comment]
Hi there,

The issue should be fixed now in version v1.10.10. I needed to change the build process somewhat, as babel needs the node_modules and babelrc file to be in the same root directory as the scripts to transpile.
